sample/kotlin/src/main/kotlin/com/datadog/android/sample/traces/OtelTracesViewModel.kt

```diff
@@ -106,6 +111,65 @@ internal class OtelTracesViewModel : ViewModel() {
         }
     }
 
+    private class ChainedContextsTask(
+        val onDone: () -> Unit
+    ) : AsyncTask<Unit, Unit, Unit>() {
+        private val tracer: Tracer = GlobalOpenTelemetry.get()
+            .getTracer("chainedContexts")
+        private val email = "john.doe@email.com"
+        private val username = "John Doe"
+        private val emailKey: ContextKey<String> = ContextKey.named("email")
+        private val usernameKey: ContextKey<String> = ContextKey.named("username")
+        private val context: Context =
+            Context.current().with(emailKey, email).with(usernameKey, username)
+        val startSpan: Span = tracer
+            .spanBuilder("submitForm")
+            .setParent(context)
+            .startSpan()
+        val scope: Scope = startSpan.makeCurrent()
```

Comment:
we could add it there also but in the end it will end up to the same thing. I just wanted to test the threading propagation here and the composed context feature.